README.md

```diff
@@ -1,2 +1,76 @@
-# idam-java-client
-Feign client for consuming idam's public API
+# Idam java client
+
+This is a client library for interacting with the idam application.
+
+## Getting started
+
+### Prerequisites
+
+- [JDK 8](https://www.oracle.com/java)
+
+## Usage
+
+Just include the library as your dependency and you will be to use the client class.
```

Comment:
Does this get published to bintray yet? If so, it would be nice to provide a url.
Or add a badge. (like here: https://github.com/hmcts/reform-api-standards)

Reply:
separate PR, i have the config locally, but waiting for this PR to go through

Reply:
kk